back/courses/models.py

```diff
@@ -1 +1,57 @@
-# Create your models here.
+from courses.models import Course
+from django.db import models
+from news.models import Post
+from social.models import Student
+
+
+class Update(models.Model):
+    date = models.DateTimeField()
+    old_course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    new_course = models.ForeignKey(Course, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.old_course.name + " : " + self.new_course.name
+
+
+class Course(models.Model):
+    name = models.CharField(max_length=100, default="Cours inconnu")
+    acronym = models.CharField(max_length=100, default="ABC")
+    department = models.CharField(
+        max_length=4, choices=Student.Department.choices, default=Student.Department.A1
```

Comment:
Is the default value mandatory? Because it is strange to put cours in a random departement by default.